### DISCUSSION TITLE
Permissions Module 'required' or not ?

### DISCUSSION THREAD

**Discussion by 2ndRecon** [John-Paul-R/Essential-Commands#168] [Q&A] [answered]
The info page says 'compatible with permission mods like . . .'

But it doesn't say whether it's required or not. Can the Commands Module work without a perms mod ?

I assume yes, since there's a toggle for API on/off, but nowhere in the docs does it say whether you 'need' a permissions mod or not.

**Comment by John-Paul-R** [comment, marked as answer]
Your assumption is correct. If the `use_permissions_api` config option is set to `false` (the default), you don't need a permissions mod.

Most commands that are reasonable for a normal player to have are granted by default when using this setting, with the others being gated behind OP.

(Notable exceptions are `nickname` and `rtp` which cannot effectively be granted to non-ops without a permissions mod, because I was silly when I initially added them. This will be changed in an upcoming major release, but that's not immediately planned.)
